### PR TITLE
Disable strong passwords by default in development

### LINKED ISSUE #1012: Should `strong_passwords` default to `false` in development?
I find myself tweaking this bit of the Bullet Train initializer a lot to turn off `strong_passwords`.

https://github.com/bullet-train-co/bullet_train/blob/e3b464b4aa7c480265b1636aa94658716192a5bc/config/initializers/bullet_train.rb#L2-L3

Would it be helpful to others to have default to being off in the development environment?

Something like:

```ruby
  config.strong_passwords = !Rails.env.development?
```

### PR DESCRIPTION
**Pull request by gazayas**
Closes #1012.

### CODE REVIEW

File: config/initializers/bullet_train.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 BulletTrain.configure do |config|
-  # Uncomment this line if you want to bypass strong passwords.
-  # config.strong_passwords = false
+  # Ensures that only strong passwords are used when registering a user.
```
Comment (jagthedrummer) [inline config/initializers/bullet_train.rb:2]:
```suggestion
  # Use strong passwords when registering a user in non-development environment.
```

Trying to make this wording a little clearer.

File: config/initializers/bullet_train.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 BulletTrain.configure do |config|
-  # Uncomment this line if you want to bypass strong passwords.
-  # config.strong_passwords = false
+  # Use strong passwords when registering a user in non-development environment.
+  # Please note that the password validations for devise are still in place
+  # when this is set to false.
```
Comment (jagthedrummer) [inline config/initializers/bullet_train.rb:4]:
```suggestion
  # in all environments.
```

**Comment by gazayas** [comment]
LGTM :+1:

**Comment by gazayas** [comment]
Just merged main, this one should be good to go.

**Pull request merged**